telemetry/validation/src/sorting.ts

```diff
@@ -0,0 +1,13 @@
+import { Metric, TelemetryField } from './telemetryDefinitions'
+
+export function fieldSort(a: TelemetryField, b: TelemetryField): number {
```

Comment:
Why is the logic spread out across many files (reorder.ts, sorting.ts)? It could all in one file. The separation isn't really gaining anything, and forces new developers to build a mental-model of all the files

Reply:
Naming is hard :)  I can consolidate them though, will take a shot at it